prospector.yaml

```diff
@@ -1,6 +1,7 @@
 strictness: medium
 test-warnings: true
 doc-warnings: true
+max-line-length: 80
 
 ignore-paths:
```

Comment:
Can remove this now. I see you remove this from my branch.

Reply:
I removed the ignore paths from your branch, I enforced this maxline length here as you set line lenth to 80 for black config

Reply:
Oh, I forgot that we still have "truelearn_experiments" in this branch.